package.json

```diff
@@ -54,7 +54,7 @@
     "superagent": "5.2.2",
     "uuid": "3.3.3",
     "verror": "1.10.0",
-    "winston": "3.2.1"
```

Comment:
you don’t need to update winston, only reinstall it to have its latest dependencies.

Reply:
Looking at it's [changelog](https://github.com/winstonjs/winston/blob/master/CHANGELOG.md), I thought it would be a good idea to update it too. f you prefer I can rollback this change.

Reply:
separation of concerns, if you don’t need to upgrade it for the vulnerability purposes you should not do it.
You don’t want to add workload on top of this work (you never know, if it will work perfectly or not)